client/src/Picks.tsx

```diff
@@ -1004,6 +1005,32 @@ function ForceBeatmapsetUpdate({ nomination }: { nomination: INomination }) {
   );
 }
 
+function visualizeDescriptionDifferences(oldDescription: string, newDescription: string): string {
+  if (oldDescription === newDescription) {
+    return newDescription;
+  }
+
+  let display = '';
```

Comment:
should build a react element/fragment/whatevr instead of html (i know thats how it was done in other parts of this app but let it be a thing of the past :100: )